virtualdns_test.go

```diff
@@ -21,8 +21,9 @@ func TestVirtualDNSUserAnalytics(t *testing.T) {
 	setup()
 	defer teardown()
 
-	since := time.Now().Add(-1 * time.Hour)
-	until := time.Now()
+	now := time.Now().UTC()
```

Comment:
This isn't related to this change and I generally recommend one discussed change per PR. Would you mind splitting this one off so it gets the proper review it deserves?

Reply:
See #349